### ISSUE TITLE
Following the same setting of "Pitfalls of GNNs", how did you obtain 1% higher accuracy than them for your teacher model?

### ISSUE THREAD

**Issue by TongLiu-github** [BUPT-GAMMA/CPF#6]
If I didnot understand wrongly, you followed the setting the same as "Pitfalls of GNNs" paper for your teacher model, but how did you obain 0.8244 for GCN on Cora, while their result is 81.5%?

**Comment by J-cabin** [comment]
The difference may stem from the fact that the DGL version of the GCN code we use is slightly different from the tensorflow version of the GCN code used in the "pitfalls" paper, as well as the difference in the model training seeds. The data partitioning we use is the same, and you can run the code to reproduce our results.

**Comment by TongLiu-github** [comment]
OK. Thanks for replying. I guess different libaraies, like Geometric, DGL, or tf, could make such a difference. But how many times do you repeat your results? 100 times averaged result like "Pitfalls" or only 1 time?

**Comment by J-cabin** [comment]
The paper shows the results of training the model with a randomly selected seed, and we have also tried training with different seeds and found that this does not affect our experimental conclusions.
